portals/publisher/src/main/webapp/source/src/app/components/GlobalPolicies/Policies/GlobalSpecificComponents/Policies.tsx

```diff
@@ -0,0 +1,610 @@
+                        Alert.success(intl.formatMessage({
+                            id: 'Policy.Mapping.Added.Successfully',
+                            defaultMessage: 'Policy mapping added successfully',
+                        }));                 
+                        history.goBack();
+                    }
+                    else {
+                        Alert.error(intl.formatMessage({
+                            id: 'Adding.Policy.Mapping.Error',
+                            defaultMessage: 'Error occurred while adding the policy mapping',
+                        })); 
+                    }                
+                })
+                .catch((/* error */) => {
+                    // console.error(error);
+                    Alert.error(intl.formatMessage({
+                        id: 'Adding.Policy.Mapping.Error',
+                        defaultMessage: 'Error occurred while adding the policy mapping',
+                    }));    
+                })
+        }
+        setLoading(false);
+    }
+
+    /**
+     * Function to update a policy mapping.
+     * Triggers if we click update button.
+     */
+    const update = () => {
+        setLoading(true);
+
+        if (validate()){
+            /**
+             * Remove UUIDs before sending to backend.
+             * If not, as backend is not expecting UUIDs, backend validation will fail.
+             */
+            const policyMapping = removeUUIDs(globalLevelPolicies);
+
+            /**
+             * Backend Call and handle the response.
+             */
+            const requestBody = {
+                "id": policyID,
+                "policyMapping": policyMapping,
+                "description": description,
+                "displayName": name,
+                "appliedGatewayLabels": appliedGatewayLabels
+            };
+            const gatewayPolicyMappingId = String(policyID);
+            const promise = API.updateGatewayPoliciesToFlows(gatewayPolicyMappingId, requestBody);
+            promise
+                .then((response) => {
+                    if (response.status === 200 || response.status === 201) {
+                        setLoading(false);
+                        Alert.success(intl.formatMessage({
+                            id: 'Policy.Mapping.Update.Success',
+                            defaultMessage: 'Policy mapping updated successfully',
+                        }));                  
+                        history.goBack();
+                    }
+                    else {
+                        Alert.error(intl.formatMessage({
+                            id: 'Policy.Mapping.Update.Error',
+                            defaultMessage: 'Error occurred while updating the policy mapping',
+                        })); 
+                    }                
+                })
+                .catch((/* error */) => {
+                    // console.error(error);
```

Comment:
```suggestion
                    
```

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)